MatchingGame.js

```diff
@@ -27,17 +25,17 @@ const getData = () => [
     { imgSrc: "images/Sasuke.jpeg", name: "Sasuke"},
 ];
 
-//Randomize
+// Randomize
 const randomize = () => {
     const cardData = getData();
     cardData.sort(() => Math.random() - 0.5);
     return cardData;
 };
 
-//Card Generator 
```

Comment:
I would elaborate more on this comment, or do not put one there. The function name is already `cardGenerator` so need for a comment with the same name basically :)